README.md

```diff
@@ -615,6 +615,92 @@ location = NestedField(LocationSerializer, return_pk=True)
     **Note:** To be able to do this you must understand how **django-restql** is implemented, specifically **DynamicFieldsMixin** class, you can check it [here](https://github.com/yezyilomo/django-restql/blob/master/django_restql/mixins.py). In fact this is how **django-restql** is implemented(just by overriding `field` method of a serializer, nothing more and nothing less).
 
 
+## View Query Control (`RestQLViewMixin`)
```

Comment:
How about using "Performance Optimization" instead of "View Query Control", it's just a suggestion.

Reply:
"Setting Up Eager Loading" makes more sense as suggested by @jheld

Reply:
I'm going with "Setting up eager loading with `EagerLoadingMixin`".